covid-nlp/main.ipynb

```diff
@@ -0,0 +1,2384 @@
+{
```

Comment:
<p>this should go to the beggining with the generic ie funcs</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/137/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>done</p>